hack/tools/release/notes.go

```diff
@@ -73,6 +73,8 @@ var (
 
 	prefixAreaLabel = flag.Bool("prefix-area-label", true, "If enabled, will prefix the area label.")
 
+	releaseCandidate            = flag.Bool("release-candidate", false, "If enabled, will add a release candidate warning header. (default false)")
+	deprecation                 = flag.Bool("deprecation", true, "If enabled, will add a templated deprecation warning header. (default false)")
```

Comment:
Comment says the default is **false** but the flag definition has it as **true**

```suggestion
	deprecation                 = flag.Bool("deprecation", false, "If enabled, will add a templated deprecation warning header. (default false)")
```